scripts/create-pv.sh

```diff
@@ -22,11 +22,32 @@ for name in $released; do
     oc patch pv -p '{"spec":{"claimRef": null}}' $name
 done
 
-NODE_NAMES=$(oc get node -o name -l node-role.kubernetes.io/worker)
-if [ -z "$NODE_NAMES" ]; then
-    echo "Unable to determine node name with 'oc' command."
-    exit 1
+# OCP 4.18+
+ssh_key="${HOME}/.crc/machines/crc/id_ed25519"
```

Comment:
This seems in line with [1][2], and I guess there's no other way than relying on the const provided by crc.

[1] https://github.com/crc-org/crc/blob/main/pkg/crc/constants/constants.go#L180
[2] https://github.com/crc-org/crc/pull/4304/